Base/QTApp/qSlicerErrorReportDialog.h

```diff
@@ -43,6 +43,7 @@ class Q_SLICER_BASE_QTAPP_EXPORT qSlicerErrorReportDialog :
 
 protected slots:
   void onLogFileOpen();
+  void onLogFileLocationOpen();
   void onLogCopy();
```

Comment:
The Build Slicer GitHub action reveals that `onLogCopy` is still present here while in the cxx file you had removed it. From @lassoan latest comment, you should probably keep the button associated with `onLogCopy` rather than replacing it.

Reply:
Both buttons were kept, so all methods are still in the `h` file.